### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by onichama**
So, I started translating this beautiful website to German. So far, the first three sections are done, but I do plan on completing this translation.

(I am native German, but I can't guarantee I didn't make any typos)

### CODE REVIEW

**Comment by hmdne** [comment]
Great progress! Once finished, I will ask someone to review this translation, to proofread and find errors, then merge.

File: translations/de/try_ruby_10.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title:  Hast du 30 Minuten Zeit? Probiere Ruby gleich aus!
+answer:
+class:  stretcher chapmark
+ok:
+error:
+---
+
+Ruby ist eine Programmiersprache aus Japan, die die Softwareentwicklung revolutioniert.
+
+Die Schönheit von Ruby liegt in seiner Ausgewogenheit zwischen Einfachheit und Kraft.
```
Comment (koffeinfrei) [inline translations/de/try_ruby_10.md]: How about "Leistungsstärke" or "Mächtigkeit". Or maybe "Fähigkeit" or just "Stärke".

File: translations/de/try_ruby_10.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Du kannst Ruby-Code in den Editor eingeben und diese Buttons zum Navigieren verwenden:
+
+- __Run__ &rarr; Führt den Code im Editor aus
+- __Copy__ &rarr; Kopiert den Beispiel-Code in den Editor
+- __Next__ &rarr; Gehe zur nächsten Lektion
```
Comment (koffeinfrei) [inline translations/de/try_ruby_10.md]: I'd use the third person for all descriptions, i.e. "Geht zur nächsten Lektion", or to stay closer to the original: "Erlaubt dir, zur nächsten Lektion zu gehen"). The same goes for "__Back__"

File: translations/de/try_ruby_10.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- __Back__ &rarr; Gehe zuürck zur vorherigen Lektion
```
Comment (koffeinfrei) [inline translations/de/try_ruby_10.md]: "zurück"

**Comment by s-gbz** [comment]
I can offer to proofread when/ if needed 😌 🤚 
Feel free to ping me (or defer to someone else, that's fine too!)

**Review by koffeinfrei — COMMENTED** [review]
Some random comments.

**Comment by hmdne** [comment]
@s-gbz 
> I can offer to proofread when/ if needed 😌 🤚 Feel free to ping me (or defer to someone else, that's fine too!)

Would be great :D I asked a friend, but probably will take some time before he has time.

File: translations/de/try_ruby_10.md (hunk 1 above)
Comment (onichama) [reply translations/de/try_ruby_10.md]: I like 'Mächtigkeit', feels closest to the English 'power' and fits together with 'Einfachheit'.

File: translations/de/try_ruby_10.md (hunk 2 above)
Comment (onichama) [reply translations/de/try_ruby_10.md]: Chose third person because I think it's easier to read.

File: translations/de/try_ruby_10.md (hunk 3 above)
Comment (onichama) [reply translations/de/try_ruby_10.md]: fixed typo

**Comment by janbiedermann** [comment]
`If `is not a Method, but that is likewise in other translations. I think thats worth fixing in all of them.

**Comment by janbiedermann** [comment]
A fun read though, fantastic!

**Comment by hmdne** [comment]
Thank you all! Sorry, my force-push caused this PR to close. I will merge it in a moment.

**Comment by hmdne** [comment]
For the record, my force-push involved a rebase (I removed the merges) and added https://github.com/ruby/TryRuby/commit/63280851eaf9495afc3c0d33520aa6383e300946 . I have published it, should be soon online!

**Comment by koffeinfrei** [comment]
Awesome :rocket: 

It seems though that not the most current version has been published at https://try.ruby-lang.org/. E.g. https://github.com/ruby/TryRuby/blob/master/translations/de/try_ruby_10.md doesn't match the first page.

**Comment by hmdne** [comment]
Thanks @koffeinfrei . I have removed the generated JSON files from the repo, seemed like we used the old thing due to them being present.

**Pull request closed without merging**